Installing-Oppia-using-Docker.md

```diff
@@ -72,49 +71,69 @@ To install Oppia under Docker, follow these steps:
 
 2. **Build Oppia for the First Time**: Run the following command to build Oppia for the first time. This step downloads all the necessary third-party libraries, python dependencies, and the other services required for the Oppia development server, which may take approximately 15-20 minutes (varies according to the network connection speed).
 
    ```
    make build
    ```
-> NOTE: This build is not related to production build by anyway. This command runs `docker compose build` under the hood, which is used to build the images for services defined in the `docker-compose.yml` file. For more information, refer to the [official documentation](https://docs.docker.com/compose/reference/build/).
+   > NOTE: The initial build process will require more time, as each layer will be built anew. However, subsequent image builds will be swifter, only recreating layers if any of them is altered. Docker's caching mechanism will be leveraged if unchanged layers are encountered, resulting in quicker subsequent builds resembling cache retrieval.
 
-3. **Start the local development server**: To start the local development server, execute the following command:
+   > NOTE: This build is not related to production build in any way. This command runs `docker compose build` under the hood, which is used to build the images for services defined in the `docker-compose.yml` file. For more information, refer to the [official documentation](https://docs.docker.com/compose/reference/build/).
+
+### Launching the development server
+
+1. **Launch the Docker Desktop app from your Applications menu**: This will start the Docker Engine, which is required to run any dockerized application or the Docker commands.
+
+2. **Start the local development server**: To start the local development server, execute the following command:
 
    ```
    make run-devserver
    ```
 
+   ![Docker Setup Terminal Logs](images/dockerSetupTerminalLogs.png)
+
+   Here are the logs that will appear in the Terminal once the development server is initiated along with all the services.
+
+   ![Docker Setup running with all services](images/dockerSetupExample.png)
+
+   You can verify the development server is up in the Containers Tab of Docker Desktop which displays that all services within `Oppia` are running.
+
    This command launches the Oppia development server, and you can continue to perform your tasks as usual.
 
-4. **Start the local development server in Offline mode**: To start the local development server in offline mode, execute the following command:
+   > NOTE: The development server requires Docker Desktop to be running, as it relies on the Docker Engine, which is initiated by Docker Desktop, to function locally.
+
+3. **Start the local development server in Offline mode**: To start the local development server in offline mode, execute the following command:
```

Comment:
This is more of an alternate path than the third step in the process of running the server. Instead of making this step 3, maybe frame it as an alternative point 2